revolv/project/tests/tests.py

```diff
@@ -242,3 +244,92 @@ class ScrapeTest(TestCase):
     def test_scrape(self):
         result = scrape.delay()
         self.assertTrue(result.successful())
+
+
+class DonationAjaxTestCase(CreateTestProjectMixin, TestUserMixin, TestCase):
+    VALIDATE = 'payment/validate'
+    SUBMIT = 'payment/submit'
+
+    def setUp(self):
+        TestUserMixin.setUp(self)
```

Comment:
I think that in general using super is the best pattern to follow. Shouldn't change functionality here at all either, but the added bonus is that if we added in another test mixin, we wouldn't have to change anything to call its `setUp` method if we have `super` in here